app/package.json

```diff
@@ -51,8 +51,8 @@
     "i18next-http-backend": "^1.4.1",
     "jest": "^28.1.0",
```

Comment:
I see some others when I run `npm outdated` - not sure if these will popup as dependabout PRs once you merge this one or not

<img width="527" alt="CleanShot 2022-10-03 at 11 11 20@2x" src="https://user-images.githubusercontent.com/371943/193648546-88f0c228-be22-4343-9afc-4a0ad151003b.png">

Reply:
Ah! Okay ran `npm update --save` and ended up updating more than just those 🤔 However everything still runs (portal storybook + tests) great so probably good! 

Motivating to consider: https://github.com/renovatebot/renovate